src/core/document.js

```diff
@@ -258,6 +258,24 @@ class Page {
     );
   }
 
+  replaceIdByRef(annotations, deletedAnnotations) {
```

Comment:
Now that we're able to use modern JavaScript also in the worker-thread, perhaps just do the following?
(The nice thing about using private fields/methods wherever possible, is that ESLint will complain if it ever becomes unused.)
```suggestion
  #replaceIdByRef(annotations, deletedAnnotations) {
```